options.go

```diff
@@ -100,3 +102,14 @@ func WithPgPlaceholders() Option {
 		return nil
 	}
 }
+
+// WithTableColumnMap provides an optional map of columns from the
+// model to the table.column name in the generated where clause
```

Comment:
Perhaps add an explicit usage example in the godoc?

Reply:
Also, we probably need a new section in the README with usage examples.

Reply:
I've added an example to the godoc comment on this function, it's a big long-winded...open to suggestions. Also added new example in README.md